umh.docs.umh.app/content/en/docs/features/devicecontainerinfrastructure/provisioning.md

```diff
@@ -1,8 +1,40 @@
 ---
 title: Provisioning
 menuTitle: Provisioning
-description:
+description: Learn the different ways to provision the the Data and the Device & Container Infrastructures.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
+
+Using the UMH's Management Console, you can provision the UMH in a simple process.
+
+
+## When should I use it?
+There are multiple ways to provision the United Manufacturing Hub. The choice depends on the use case and the available resources.
+
+- **Install UMH + Operating System**: This option is currently not implemented. You will get a .iso image that installs the corresponding operating system and the UMH. It also configures the Management Companion for you automatically. 
+- **Install UMH Only**: If you want to install the UMH on your server, this option is the best choice. This option requires that a supported operating system is installed on your server. The user will receive a single command that must be executed via SSH on an existing Linux system. You find the software and hardware requirements on the [Installation](/docs/getstarted/installation/) page.
+- **Test the UMH**: This option is currently not implemented. You will get a local installation guide on Linux OS and Windows.
```

Comment:
- Put the "install umh only" option first, as it's the only one implemented
- Use inline code for `.iso`
- "You find the software and hardware..." should say "You can find..."